oap-server/server-starter/src/main/resources/ui-initialized-templates/postgresql/postgresql-service.json

```diff
@@ -31,7 +31,7 @@
                     "title": "Fetched Rows Trend (rows per second)"
                   },
                   "expressions": [
-                    "meter_pg_fetched_rows_rate"
+                    "aggregate_labels(meter_pg_fetched_rows_rate,sum(service_instance_id))"
```

Comment:
Do you want to show instance metrics separately?

Reply:
This is service dashboard, you should merge instance tag to aggregate.

Reply:
Yes, I put details in instance level, and service only display different instances metrics.

Reply:
> Yes, I put details in instance level, and service only display different instances metrics.

This isn't the point of this fix. The key to do this is making service metric back. If only showing per instance, the old kernel works already.

Please fix accordingly.